docs/extend-graphql-schema.md

```diff
@@ -3,4 +3,568 @@ id: extend-graphql-schema
+Try to execute the query below in Graph*i*QL:
+
+```
+{
+  physicalStores {
+    address {
+      street
+      city
+      country
+      postcode
+    }
+    coordinates {
+      latitude
+      longitude
+    }
+    openingHours {
+      day
+      openingTimes {
+        from {
+          h
+          m
+        }
+        to {
+          h
+          m
+        }
+      }
+    }
+  }
+}
+```
+
+Our GraphQL module does not yet have any code for sending content, so you must
+see the following response:
+
+```json
+{
+  "data": {
+    "physicalStores": null
+  }
+}
+```
+
+Let’s now add some executable logic to our module.
+
+### Implement resolvers
+
+The GraphQL middleware relies on resolver functions to determine the data to
+return for a given field. This is where most of the « real work » is done, for
+instance by fetching remote datasources or transforming data.
+
+Resolvers are exposed using the `resolvers` key of the module definition. It
+should be a « resolver map »: an object where each key is a GraphQL type name,
+and values are mapping between field names and resolver function. Resolver
+functions may return a
+[Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises)
+for asynchronous operations.
+
+> To know more about resolvers and their internals, we recommend the reading of
+> [GraphQL Tools documentation](https://www.apollographql.com/docs/graphql-tools/resolvers).
+
+First, update the `src/server/modules/physical-stores/index.js` module
+definition as follow:
+
+```diff
+import typeDefs from "./schema.gql";
++import resolvers from "./resolvers";
+
+export default {
+  namespace: "PhysicalStores",
+-  typeDefs: typeDefs
++  typeDefs: typeDefs,
++  resolvers: resolvers
+};
+```
+
+And then create the `src/server/modules/physical-stores/resolvers.js` file with
+a resolver map as below (we are going to analyze it in details in a few
+seconds):
+
+```js
+import axios from "axios";
+
+export default {
+  Query: {
+    physicalStores: () => findAllStores()
+  },
+
+  PhysicalStore: {
+    coordinates: store => ({
+      latitude: store.lat,
+      longitude: store.lon
+    }),
+    openingHours: () => generalOpeningHours
+  },
+
+  Time: {
+    h: time => time.split(":")[0],
+    m: time => time.split(":")[1]
+  },
+
+  PostalAddress: {
+    street: address => address.road
+  }
+};
+
+const findAllStores = () =>
+  axios
+    .get("https://nominatim.openstreetmap.org/", {
+      params: {
+        format: "json",
+        countrycodes: "fr",
+        addressdetails: 1,
+        q: "foot locker", // for instance…
+        limit: 10
+      }
+    })
+    .then(response => response.data);
+
+const generalOpeningHours = [
+  "TUESDAY",
+  "WEDNESDAY",
+  "THURSDAY",
+  "FRIDAY",
+  "SATURDAY"
+].map(dayOfWeek => ({
+  day: dayOfWeek,
+  openingTimes: [{ from: "9:00", to: "12:30" }, { from: "14:00", to: "20:00" }]
+}));
+```
+
+Let’s analyze this code part by part to understand what is possible to achieve
```

Comment:
That's not really how things are explained in the other documentation pages. We generally went for a step by step approach : first let's do this - ah, this is missing, let's do this next - and so on.

That would be too big of a change right now, but I'll try to improve this part later if your ok with this.